sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/HPEArchitect.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  Image,
+  PageContent,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import { Metric } from '../../../../components';
+
+export const HPEArchitect = ({ size }) => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  const columns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : ['3/4', 'auto'];
+  const gap = {
+    xsmall: 'medium',
+    small: 'medium',
+    medium: 'medium',
+    large: 'large',
+    xlarge: 'large',
+  };
+
+  const metricColumns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : { count: 3, size: 'auto' };
+
+  const areasDefault = [['quotes', 'hpeBook']];
+
+  const areasSmall = [
+    ['quotes', 'unassigned'],
+    ['hpeBook', 'unassigned'],
+  ];
+
+  const areas = ['xsmall', 'small'].includes(breakpoint)
+    ? areasSmall
+    : areasDefault;
+
+  return (
+    <PageContent
+      pad="large"
+      background={{
+        fill: 'horizontal',
+        // this is the closet color we have to what they have #00384D
+        color: 'graph-1',
```

Comment:
Semantically wouldn't be correct to use a datavis color here. Let's note this as a "gap" in background colors.